build.yaml

```diff
@@ -3931,3 +3931,26 @@ steps:
     dependsOn:
       - ci_utils_image
       - default_ns
+  - kind: runImage
+    name: set_gcp_ar_cleanup_policies
+    resources:
+      memory: standard
+      cpu: '0.25'
+    image:
+      valueFrom: ci_utils_image.image
+    scopes:
+      - deploy
```

Comment:
I think it's okay to add `- dev` here. Then you could test it with a dev deploy.

Reply:
I should have highlighted this in the PR, my bad.

I discussed this with @daniel-goldstein and he suggested it might be slightly dangerous to do this on dev deploy. 

As I recall (and may well be wrong), we use one aritfact registry for both dev and production. If one of us makes an experimental change to those cleanup policies, it'll affect everything.

Reply:
ok. That makes sense. I'll look over it one more time then more carefully. If it fails on the next deploy, it's not a big deal and we can fix any issues then.

Reply:
My biggest concern is it will fail due to the credentials not being exposed in the job for gcloud to use. Do you have an example of the output with the changes?

Reply:
Output:
```json
[
  {
    "action": {
      "type": "DELETE"
    },
    "condition": {
      "olderThan": "2592000s",
      "tagPrefixes": [
        "cache-"
      ],
      "tagState": "TAGGED"
    },
    "name": "delete_cache"
  },
  {
    "action": {
      "type": "DELETE"
    },
    "condition": {
      "olderThan": "259200s",
      "tagPrefixes": [
        "dev-"
      ],
      "tagState": "TAGGED"
    },
    "name": "delete_dev"
  },
  {
    "action": {
      "type": "DELETE"
    },
    "condition": {
      "olderThan": "604800s",
      "tagPrefixes": [
        "cache-pr-"
      ],
      "tagState": "TAGGED"
    },
    "name": "delete_pr_cache"
  },
  {
    "action": {
      "type": "DELETE"
    },
    "condition": {
      "olderThan": "259200s",
      "tagPrefixes": [
        "test-deploy-"
      ],
      "tagState": "TAGGED"
    },
    "name": "delete_test_deploy"
  },
  {
    "action": {
      "type": "DELETE"
    },
    "condition": {
      "olderThan": "259200s",
      "tagPrefixes": [
        "test-pr-"
      ],
      "tagState": "TAGGED"
    },
    "name": "delete_test_pr"
  },
  {
    "action": {
      "type": "DELETE"
    },
    "condition": {
      "tagState": "UNTAGGED"
    },
    "name": "delete_untagged"
  },
  {
    "action": {
      "type": "KEEP"
    },
    "mostRecentVersions": {
      "keepCount": 10,
      "packageNamePrefixes": [
        "admin-pod",
        "auth",
        "base",
        "batch",
        "batch-worker",
        "blog_nginx",
        "ci",
        "ci-hello",
        "ci-utils",
        "create_certs_image",
        "curl",
        "git-make-bash",
        "gpu",
        "hail-benchmark",
        "hail-buildkit",
        "hail-dev",
        "hail-run",
        "hail-ubuntu",
        "hail-ubuntu-py-3-10",
        "hail-ubuntu-python-3-10",
        "hail-ubuntu-python-3-11",
        "hailgenetics/hail",
        "hailgenetics/hailtop",
        "hailgenetics/vep-grch37-85",
        "hailgenetics/vep-grch38-95",
        "linting",
        "monitoring",
        "netcat",
        "query-build",
        "test-ci-utils",
        "test_hello_create_certs_image",
        "volume",
        "website",
        "workdir"
      ]
    },
    "name": "keep_most_recent_deploy"
  },
  {
    "action": {
      "type": "KEEP"
    },
    "condition": {
      "packageNamePrefixes": [
        "alpine",
        "debian",
        "envoyproxy/envoy",
        "ghost",
        "google/cloud-sdk",
        "grafana/grafana",
        "jupyter/scipy-notebook",
        "moby/buildkit",
        "python",
        "redis",
        "ubuntu"
      ],
      "tagState": "ANY"
    },
    "name": "keep_third_party"
  }
]
```